hammers/scripts/unutilized_lease_reaper.py

```diff
@@ -0,0 +1,178 @@
+# coding: utf-8
+'''
+.. code-block:: bash
+
+    unutilized-leases {info, delete}
+
+Deletes leases that do not launch any instances after a period of time
+specified by the admin.
+
+* ``info`` to just display leases or actuall delete them with ``delete``
+'''
+import argparse
+from collections import defaultdict
+from datetime import datetime, timedelta
+from pprint import pprint
+from pytz import timezone
+import sys
+
+from hammers import osapi
+from hammers.slack import Slackbot
+from hammers.notifications import _email
+from hammers.osrest import blazar, ironic, keystone
+
+DEFAULT_WARN_HOURS = 6
+DEFAULT_GRACE_HOURS = 9
+
+
+def parse_time(time):
+    time = time.split('+')[0].split('.')[0]
+    dt_fmt = '%Y-%m-%dT%H:%M:%S'
+    return datetime.strptime(time, dt_fmt).replace(tzinfo=timezone('UTC'))
+
+
+def inviolation_filter(hour):
+    now = datetime.utcnow().replace(tzinfo=timezone('UTC'))
```

Comment:
I don't think you need to replace timezone if you're using `utcnow`

Reply:
It throws an error with out it and says the two can't be compared. I think it's because pytz handles timezones differently that datetime.